tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -220,6 +373,18 @@ async def verify_replicas_are_in_wedged_checkpoint(self, bft_network, previous_c
                         if checkpoint_after == previous_checkpoint + 2:
                             break
 
+    async def _get_metric(self, replica_id, bft_network, mtype, mname):
+        with trio.fail_after(seconds=30):
+            while True:
+                with trio.move_on_after(seconds=1):
+                    try:
+                        key = ['replica', mtype, mname]
+                        value = await bft_network.metrics.get(replica_id, *key)
+                    except KeyError:
+                        # metrics not yet available, continue looping
+                        print(f"KeyError! '{mname}' not yet available.")
+                    else:
+                        return value
```

Comment:
this looks like generic logic that can live in `bft_network.py`

Reply:
done